docs/rfcs/0001-designing-in-the-open.md

```diff
@@ -0,0 +1,139 @@
+---
+title: "Designing in the Open"
+weight: 1
+---
+
+# Designing in the Open
+
+* Date: 2021-11-02
+* Author: Robert Fratto (@rfratto)
+* PR: [grafana/agent#1055](https://github.com/grafana/agent/pull/1055)
+
+## Summary
+
+Traditionally, design proposals to Grafana Agent have been done in private by
+default, and then only made public after internal consensus was reached.
+This causes a few problems:
+
+* Community concerns are less likely to have an impact as core maintainers have
+  already agreed on a solution.
+* Initial design is gated to Grafana Labs employees.
+* Historical design docs with context and discussions become hard to find both
+  internally and even harder publicly.
+
+In the spirit of building a broader community, this document proposes inverting
+the design process to be public by default.
+
+## Goals
+
+* Outline a process to support public-first design documents
+* Lower the barrier to entry to becoming a maintainer that is not employed at
+  Grafana Labs
+* Encourage a community to contribute input to design documents
```

Comment:
This document already has some form of guidelines on how to deal with RFCs. What I'm missing is how an RFC can get accepted. Is this the document for it to clarify that or should that become a follow-up?

Reply:
Sure, we can clarify that proposals are up to the discretion/consensus of the governance team. While that's all Grafana Labs employees at the moment, I'm keeping a close eye out for external collaborators that want to be involved with the project at that level.